examples/plot_benchmark_DL.py

```diff
@@ -0,0 +1,98 @@
+"""
+====================================================================
+Benchmarking with MOABB with DeepLearning architecture in Tensorflow
+====================================================================
+This example shows how to use MOABB to benchmark a set of DeepLearning pipeline (Tensorflow)
+on all available datasets.
+For this example, we will use only one dataset to keep the computation time low, but this benchmark is designed
+to easily scale to many datasets.
+"""
+# Authors: Igor Carrara <igor.carrara@inria.fr>
+#
+# License: BSD (3-clause)
+
+import os
+import random
+
+import matplotlib.pyplot as plt
+import numpy as np
+import tensorflow as tf
+from tensorflow import keras
+
+from moabb import benchmark, set_log_level
+from moabb.analysis.plotting import score_plot
+
+
+set_log_level("info")
+
+import absl.logging
+
+
+# Avoid output Warning
+absl.logging.set_verbosity(absl.logging.ERROR)
+os.environ["TF_CPP_MIN_LOG_LEVEL"] = "3"
+
+# Print Information Tensorflow
+print(f"Tensorflow Version: {tf.__version__}")
+print(f"Keras Version: {keras.__version__}")
+
+CPU = len(tf.config.list_physical_devices("CPU")) > 0
+print("CPU is", "AVAILABLE" if CPU else "NOT AVAILABLE")
+
+GPU = len(tf.config.list_physical_devices("GPU")) > 0
+print("GPU is", "AVAILABLE" if GPU else "NOT AVAILABLE")
+
+# Allow parallelization on all physical devices available
+mirrored_strategy = tf.distribute.MirroredStrategy()
+
+
+# Set up reproducibility of Tensorflow
+def setup_seed(seed):
+    random.seed(seed)
+    np.random.seed(seed)
+    tf.random.set_seed(seed)  # tf cpu fix seed
+    os.environ[
+        "TF_DETERMINISTIC_OPS"
+    ] = "1"  # tf gpu fix seed, please `pip install tensorflow-determinism` first
+
+
+setup_seed(42)
```

Comment:
I think it good to move this function to the utils to use also in the braindecode pipeline, or use a function from tensorflow to set determinism:  

```suggestion

tf.keras.utils.set_random_seed(seed)

```